qiita_db/support_files/patches/python_patches/51.py

```diff
@@ -491,8 +527,8 @@ def transfer_job(analysis, command_id, params, input_artifact_id, job_data,
         (sum_taxa_cmd_id, 'sort', 'bool', False, 'False'),
         # Beta Diversity
         (bdiv_cmd_id, 'tree', 'string', False, ''),
-        (bdiv_cmd_id, 'metrics',
-         'mchoice:["abund_jaccard","binary_chisq","binary_chord",'
+        (bdiv_cmd_id, 'metric',
```

Comment:
Yes it is intentional, I'm making the command in Qiita to be per-metric based as this will be easier to show the results